Jamulus.pro

```diff
@@ -93,10 +93,29 @@ win32 {
 
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
-        message(Using Jack instead of ASIO.)
-
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "i386") {
+            !exist("C:/Program Files (x86)") {
+                message ("Cross compilation build")
+                programfilesdir = "C:/Program Files (x86)"
+                !exists("$${"programfilesdir"}/JACK2/include/jack/jack.h") {
+                    message("Warning: jack.h was not found at the usual place, maybe JACK2 32 bits is not installed to cross compile.")
+                }
```

Comment:
It looks like the `exists` block is required in all three code paths. Also, it seems to be identical as variables are used (which is good :)). Maybe this can be condensed to a single check after the last block?

`message("Warning: jack.h was not found in the expected location ($${programfilesdir}). Ensure that the right JACK2 variant is installed (32bit vs. 64bit).")`

Reply:
The check is identical but the message really different. Don't know if it is better like that.

Reply:
9a5c412a